core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -70,6 +70,8 @@ private[spark] class AppStatusListener(
   private val liveTasks = new HashMap[Long, LiveTask]()
   private val liveRDDs = new HashMap[Int, LiveRDD]()
   private val pools = new HashMap[String, SchedulerPool]()
+
+  private val SQL_EXECUTION_ID_KEY = "spark.sql.execution.id"
```

Comment:
Here we can't access `SQLExecution.EXECUTION_ID_KEY`. So use a variable for the key string.